docs/advanced/document-store/qdrant.md

```diff
@@ -79,13 +80,19 @@ The following configs can be set:
 | `collection_name`     | Qdrant collection name client                                                                                                                | **Random collection name generated**             |
-| `host`                | Hostname of the Qdrant server                                                                                                                | 'localhost'                                      |
-| `port`                | port of the Qdrant server                                                                                                                   | 6333                                             |
-| `distance`            | Distance metric to be used during search. Can be 'cosine', 'dot' or 'euclidean'                                                              | 'cosine'                                         |
-| `scroll_batch_size`   | batch size used when scrolling over the storage                                                                                              | 64                                               |
-| `ef_construct`        | Number of neighbours to consider during the index building.  Larger the value - more accurate the search, more time required to build index. | `None`, defaults to the default value in Qdrant* |
-| `full_scan_threshold` | Minimal amount of points for additional payload-based indexing.                                                                              | `None`, defaults to the default value in Qdrant*                                               |
-| `m`                   | Number of edges per node in the index graph. Larger the value - more accurate the search, more space required.                               | `None`, defaults to the default value in Qdrant*                                               |
+| `distance`            | Distance metric to be used during search. Can be 'cosine', 'dot' or 'euclidean'                                                              | `'cosine'`                                       |
+| `host`                | Hostname of the Qdrant server                                                                                                                | `'localhost'`                                    |
+| `port`                | Port of the Qdrant server                                                                                                                    | `6333`                                           |
+| `grpc_port`           | Port of the Qdrant gRPC interface                                                                                                            | `6334`                                           |
+| `prefer_grpc`         | Set `true` to use gPRC interface whenever possible in custom methods                                                                         | `False`                                          |
```

Comment:
```suggestion
| `prefer_grpc`         | Set `True` to use gPRC interface whenever possible in custom methods                                                                         | `False`                                          |
```

Reply:
It should be capitalized, right? If so, fix this throughout the doc

Reply:
yeah sure :)